add rockspec.

xpol opened the pull request:
I have uploaded this rockspec to luarocks.org.
notify me if you want to upload it by yourself.
I can delete the rock on luarocks.org then.

xpol committed:
add rockspec.

BlackBulletIV commented:
Haven't used luarocks myself before. I'm guessing the rockspec on their servers would need to be updated with every version? If that's the case it may be better for me to upload it myself.

xpol commented:
Yes the rockspec contents and filename needs to be changed time to time.

xpol committed:
prevent module been loaded more than once.

for some reason, e.g. apply hot update, user may need to clean package.loaded.
but load strong more than once will cause endless loop.

xpol committed:
Merge branch 'prevent-load-twice'
